### PR TITLE
Fix SSR

### PR DESCRIPTION
**Pull request by mondeja**

### CODE REVIEW

File: leptos_hotkeys/Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,6 @@ keywords = ["leptos", "hotkeys", "wasm"]
 leptos = { version = "0.6.5", features = ["nightly"] }
 wasm-bindgen = { version = "0.2.92" }
 log = "0.4.20"
-cfg-if = { version = "1.0.0", features = [] }
```
Comment (friendlymatthew) [inline leptos_hotkeys/Cargo.toml:16]: amazing

File: leptos_hotkeys/src/use_hotkeys.rs [hunk 2]
```diff
@@ -41,47 +42,58 @@ fn is_hotkey_match(hotkey: &Hotkey, pressed_keyset: &mut HashMap<String, web_sys
...
-    key_combination: String,
-    on_triggered: Callback<()>,
-    scopes: Vec<String>,
+    #[cfg_attr(feature = "ssr", allow(unused_variables))] key_combination: String,
+    #[cfg_attr(feature = "ssr", allow(unused_variables))] on_triggered: Callback<()>,
+    #[cfg_attr(feature = "ssr", allow(unused_variables))] scopes: Vec<String>,
 ) {
-    let parsed_keys: HashSet<Hotkey> = key_combination.split(',').map(Hotkey::new).collect();
+    #[cfg(not(feature = "ssr"))]
+    {
```
Comment (mondeja) [inline leptos_hotkeys/src/use_hotkeys.rs:50]: You don't need `cfg_if` if you wrap a block like this.

**Review by friendlymatthew — APPROVED** [review]
Looks great, but one quick question:

why do we add the `feature = "ssr"` for the parameters of `use_hotkeys_scoped` and others?

File: leptos_hotkeys/src/use_hotkeys.rs [hunk 3]
```diff
@@ -41,47 +42,58 @@ fn is_hotkey_match(hotkey: &Hotkey, pressed_keyset: &mut HashMap<String, web_sys
 }
 
 pub fn use_hotkeys_scoped(
-    key_combination: String,
-    on_triggered: Callback<()>,
-    scopes: Vec<String>,
+    #[cfg_attr(feature = "ssr", allow(unused_variables))] key_combination: String,
```
Comment (mondeja) [inline leptos_hotkeys/src/use_hotkeys.rs:45]: Change this:
```suggestion
    key_combination: String,
```

Execute `cargo make lint`:

```
error: unused variable: `key_combination`
  --> leptos_hotkeys/src/use_hotkeys.rs:45:5
   |
45 |     key_combination: String,
   |     ^^^^^^^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_key_combination`
   |
   = note: `-D unused-variables` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(unused_variables)]`

error: could not compile `leptos_hotkeys` (lib) due to 1 previous error
```

Clippy complains about not used when ssr feature active.

Comment (mondeja) [reply leptos_hotkeys/src/use_hotkeys.rs:45]: Note that is not something that affetcs the code, really is a `clippy()` comment.

Comment (friendlymatthew) [reply leptos_hotkeys/src/use_hotkeys.rs:45]: Sounds good! Everything looks great! Really clever with the std::collections::Hashmap :)

**Pull request merged**